plugins/@grouparoo/hubspot/__tests__/fixtures/destination-mapping-options.js

```diff
@@ -0,0 +1,108 @@
oritedOrder":-1,"displayMode":"current_value","optionsAreMutable":null,"displayOrder":22,"deleted":null,"textDisplayHint":null,"updatedUserId":null,"readOnlyDefinition":false,"referencedObjectType":null,"searchableInGlobalSearch":false,"hasUniqueValue":false,"options":[]},{"name":"industry","label":"Industry","description":"The industry a contact is in","groupName":"contactinformation","type":"string","fieldType":"text","createdAt":1565059269345,"createdUserId":null,"calculated":false,"isCustomizedDefault":false,"updatedAt":1565384992959,"externalOptions":false,"showCurrencySymbol":null,"optionSortStrategy":null,"numberDisplayHint":null,"hidden":false,"currencyPropertyName":null,"readOnlyValue":false,"hubspotDefined":true,"searchTextAnalysisMode":null,"formField":true,"mutableDefinitionNotDeletable":true,"favorited":false,"favoritedOrder":-1,"displayMode":"current_value","optionsAreMutable":null,"displayOrder":23,"deleted":null,"textDisplayHint":null,"updatedUserId":null,"readOnlyDefinition":false,"referencedObjectType":null,"searchableInGlobalSearch":false,"hasUniqueValue":false,"options":[]},{"name":"associatedcompanyid","label":"Associated Company ID","description":"HubSpot defined ID of a contact's associated company in HubSpot.","groupName":"contactinformation","type":"number","fieldType":"number","createdAt":1565059269377,"createdUserId":null,"calculated":false,"isCustomizedDefault":false,"updatedAt":1603374711956,"externalOptions":false,"showCurrencySymbol":null,"optionSortStrategy":null,"numberDisplayHint":"unformatted","hidden":true,"currencyPropertyName":null,"readOnlyValue":false,"hubspotDefined":true,"searchTextAnalysisMode":null,"formField":false,"mutableDefinitionNotDeletable":false,"favorited":false,"favoritedOrder":-1,"displayMode":"current_value","optionsAreMutable":null,"displayOrder":24,"deleted":null,"textDisplayHint":null,"updatedUserId":null,"readOnlyDefinition":true,"referencedObjectType":"COMPANY","searchableInGlobalSearch":false,"hasUniqueValue":false,"options":[]},{"name":"associatedcompanylastupdated","label":"Associated Company Last Updated","description":"This field is meaningless on its own, and is solely used for triggering dynamic list updates when a company is updated","groupName":"contactinformation","type":"number","fieldType":"number","createdAt":1565059269415,"createdUserId":null,"calculated":false,"isCustomizedDefault":false,"updatedAt":1571085943940,"externalOptions":false,"showCurrencySymbol":null,"optionSortStrategy":null,"numberDisplayHint":null,"hidden":true,"currencyPropertyName":null,"readOnlyValue":true,"hubspotDefined":true,"searchTextAnalysisMode":null,"formField":false,"mutableDefinitionNotDeletable":false,"favorited":false,"favoritedOrder":-1,"displayMode":"current_value","optionsAreMutable":null,"displayOrder":25,"deleted":null,"textDisplayHint":null,"updatedUserId":null,"readOnlyDefinition":true,"referencedObjectType":null,"searchableInGlobalSearch":false,"hasUniqueValue":false,"options":[]}], [
```

Comment:
I think we need `prettier` here.
@evantahler know why this isn't failing the `lint`